docs/docs/testing.md

```diff
@@ -1131,9 +1131,9 @@ export default NameForm
 
 Now, we can extend the `test` file which Redwood generated. We're going to want to:
 
-1) Import `waitFor` from the `@redwoodjs/testing/web` library.
-2) Add an import to `@testing-library/user-event` for its `default`.
-3) Provide an `onSubmit` prop to our "renders successfully" test.
+1. Import `waitFor` from the `@redwoodjs/testing/web` library.
+2. Add an import to `@testing-library/user-event` for its `default`.
+3.Provide an `onSubmit` prop to our "renders successfully" test.
```

Comment:
two and three here are on the same line:

<img width="865" alt="image" src="https://github.com/redwoodjs/redwood/assets/32992335/2d2bfe20-5882-4d5c-b4d7-654210d269bd">

i think it's because of the missing space after the period:

```suggestion
3. Provide an `onSubmit` prop to our "renders successfully" test.
```

Reply:
ugh... so close to get in just one go. thanks dom.